.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -53,8 +67,12 @@ jobs:
       - name: Deploy Standalone Plugin - ${{ matrix.slug }}
         uses: 10up/action-wordpress-plugin-deploy@stable
         env:
-          SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
-          SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          # TODO Once the workflow is tested, we will remove the comment and use the secret SVN access.
+          #SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          #SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          # TODO Once the workflow is tested, we will remove this test credential.
+          SVN_PASSWORD: SVN_PASSWORD
+          SVN_USERNAME: SVN_USERNAME
```

Comment:
Before merging, please make sure to resolve these comments and bring back the usage of the `secrets`.

Reply:
Updated in https://github.com/WordPress/performance/pull/710/commits/52c6eb1084b33da32d8ff2fdd720132e221b317c